larray/tests/test_session.py

```diff
@@ -7,7 +7,8 @@
 import pandas as pd
 import pytest
 
-from larray.tests.common import (assert_array_nan_equal, inputpath, tmp_path, meta,
+from larray.tests.common import meta            # noqa: F401
```

Comment:
why do we need to import meta if it is unused?

Reply:
`meta` is a [pytest fixture](https://docs.pytest.org/en/stable/fixture.html#fixtures-as-function-arguments) (`meta` is defined in the /tests/common.py module) and is actually used as argument in many unittests. 
It's just that flake8 is confused with pytest fixtures.